editor/editor_node.cpp

```diff
@@ -6631,13 +6631,13 @@ void EditorNode::_bottom_panel_raise_toggled(bool p_pressed) {
 void EditorNode::_update_renderer_color() {
 	String rendering_method = renderer->get_selected_metadata();
 
-	if (rendering_method == "forward_plus") {
+	if (rendering_method.begins_with("forward_plus")) {
```

Comment:
Why `begins_with`?

Reply:
Oh sorry I forget to delete that, at first I set the metadata with a suffix too.